oat-sa/tao-core
# feat/AUT-2332 add extra services method to check explicit show

Related to https://oat-sa.atlassian.net/browse/AUT-2332
**Warning:** This PR is targeted to [feat/AUT-2332/add-direction-languages-endpoint](https://github.com/oat-sa/tao-core/tree/feat/AUT-2332/add-direction-languages-endpoint) that contains v2 languages API

<img width="1902" alt="Screenshot 2022-07-12 at 20 57 18" src="https://user-images.githubusercontent.com/2067027/178714821-abc84ca1-ce5e-4254-a90a-d1d9f5ad2545.png">

**Description:**
Add languages to inline CKE Editor

**Changes:**
* Added language plugin for CKE editor, disable if feature flag is not provided (currently in debugging state to show always).

**How to check:** 
* Install 3 PRS of extension below and modify 2 links to NPM packages
* Create any Item in authoring
* Click on the description part of the text editor
* Observe a new Set language button with a list
* Select text and change language for the segment

**Requires:**
* BE & CKE base part: https://github.com/oat-sa/tao-core/pull/3575
* Manually change `package.json` in tao/views
    - "@oat-sa/tao-core-shared-libs": "https://github.com/oat-sa/tao-core-shared-libs-fe#feat/AUT-2332_add-language-to-editor",
    - "@oat-sa/tao-core-ui": "https://github.com/oat-sa/tao-core-ui-fe#feat/AUT-2332_add-language-to-editor",
* Feature flag: https://github.com/oat-sa/extension-tao-deliver-connect/pull/128
* Item QTI: (https://github.com/oat-sa/extension-tao-itemqti/pull/2170)
* Media manager:  https://github.com/oat-sa/extension-tao-mediamanager/pull/409


**Others PR's**
Core UI: https://github.com/oat-sa/tao-core-ui-fe/pull/471
Core Shared libs: https://github.com/oat-sa/tao-core-shared-libs-fe/pull/25
CKE editor: https://github.com/oat-sa/ckeditor-dev/pull/29

Part 1/6
